src/main/scala/org/ergoplatform/mining/AutolykosPowScheme.scala

```diff
@@ -21,13 +23,16 @@ import scala.util.Try
 /**
   * Autolykos PoW puzzle scheme reference implementation.
   *
+  * Based on k-sum problem, so general idea is to find k numbers in a table of size N, such that
+  * sum of numbers (or a hash of the sum) is less than target value.
+  *
   * See https://docs.ergoplatform.com/ErgoPow.pdf for details
   *
-  * Mining process is implemented in inefficient way and should not be used in real environment.
+  * CPU Mining process is implemented in inefficient way and should not be used in real environment.
   *
   * @see papers/yellow/pow/ErgoPow.tex for full description
   * @param k - number of elements in one solution
-  * @param n - power of number of elements in a list
+  * @param n - initially, N = 2^^n
```

Comment:
Why `^^` and not just `2^n`?

Reply:
@oskin1 getting "Unclosed tag" warning with 2^n just (in IDEA)

Reply:
@kushti , anyway it's a bit misleading